scss/_vars.scss

```diff
@@ -7,6 +7,10 @@ $color-secondary: var(--secondary, #417690);
 $color-accent: var(--accent, #f5dd5d);
 $color-primary-fg: var(--primary-fg, #fff);
 
+/* admin-index specific variables */
+// default of 10px is the padding-bottom default of the header
+$djai-dropdown-menu-spacing-top: var(--djai-dropdown-menu-spacing-top, 10px);
```

Comment:
I suppose in this case we could set the default CSS variable values, but the order of stylesheet inclusion (admin-index is loaded in the body!) makes overriding in your own custom stylesheet harder than it should be.

This way, the CSS var is used if it is defined, otherwise the default value fallback is used. And to not have to repeat the fallback value everywhere, I've opted to wrap that in a sass variable.